lib/hm_crypto/error_container.ex

```diff
@@ -93,6 +100,7 @@ defmodule HmCrypto.ErrorContainer do
            nonce: nonce,
            encrypted_flag: encrypted_flag,
            command_binary: command
+           # content_type: content_type
```

Comment:
Should we set it explicitly to `nil` or `:unknown` here?

Reply:
it's set to `:unknown` since this is the error has this type

https://github.com/highmobility/hm-crypto-elixir/blob/telematics-v2-support-with-content-type/lib/hm_crypto/error_container.ex#L39